### DISCUSSION TITLE
Why can't I use same user on multiple device?

### DISCUSSION THREAD

**Discussion by theprantadutta** [datalust/seq-tickets#2178] [General]
Hi, thank you for creating this awesome tool, and I really like what I do here, now, I want to ask why can't I login to the same account from different devices.
I have a home PC, and I mostly work from there, but when I have to access the seq dashboard from PC, it logs me out from other devices, why?
Is there any way I can enable it or something?
Once again, thank you for all your hard work.

**Comment by liammclennan** [comment]
Hi @theprantadutta 

That is a restriction on the free 'Individual' license. The other subscription options do not have this restriction: https://datalust.co/pricing

**Reply by theprantadutta** [reply]
Oh, okay, got it.
